packages/app-project/src/components/AuthModal/shared/components/withCustomFormik/withCustomFormik.spec.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { mount, shallow } from 'enzyme'
+import { expect } from 'chai'
+import { Box, Button, FormField, TextInput } from 'grommet'
+import sinon from 'sinon'
+import withCustomFormik from './withCustomFormik'
+
+const MockForm = ({ handleBlur, handleChange, handleSubmit, values }) => (
+  <Box as='form' onSubmit={handleSubmit}>
+    <FormField htmlFor='testInput' label='Enter text' >
+      <TextInput
+        id='testInput'
+        name='testInput'
+        onBlur={handleBlur}
+        onChange={handleChange}
+        type='text'
+        value={values.testInput}
+      />
+    </FormField>
+    <Button type='submit' label='Submit' />
+  </Box>
+)
+
+const WrappedMockForm = withCustomFormik(MockForm)
+
+const initialValues = { testInput: '' }
+const onSubmit = sinon.spy()
+
+describe('Higher Order Component > withCustomFormik', function () {
+  it('renders without crashing', function () {
+    const wrapper = shallow(<WrappedMockForm initialValues={initialValues} onSubmit={onSubmit} />)
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should pass Formik required props', function () {
+    const wrapper = shallow(
+      <WrappedMockForm
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+      />
+    )
+
+    const formikHOC = wrapper.find('FormikHOC')
+
+    expect(formikHOC.props().initialValues).to.equal(initialValues)
+    expect(formikHOC.props().onSubmit).to.equal(onSubmit)
+  })
+
+  it('should pass Formik optional props', function () {
+    const validate = sinon.spy()
+    const wrapper = shallow(
+      <WrappedMockForm
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+        validate={validate}
+      />
+    )
+    const formikHOC = wrapper.find('FormikHOC')
+
+    expect(formikHOC.props().validate).to.equal(validate)
+  })
+
+  it('should call onChange prop', function () {
+    const onChangeStub = sinon.stub()
+
+    const wrapper = mount(
+      <WrappedMockForm
+        initialValues={initialValues}
+        onChange={onChangeStub}
+        onSubmit={onSubmit}
+      />
+    )
+    const formikProps = wrapper.find(MockForm).props()
```

Comment:
Out of interest, why did we have to use `wrapper.dive().find('MockForm')` to get the form before, but `wrapper.find(MockForm)` works here?

Reply:
Oh wait, we were using shallow mounting before and now use a full render?

Reply:
Yep